### DISCUSSION TITLE
ComboBox dropdown alignment

### DISCUSSION THREAD

**Discussion by DarkMortainius** [AvaloniaUI/Avalonia#15660] [Q&A] [answered]
I have a ComboBox and I'd like all the text (including the dropdown box) to be left aligned. For some reason, I can't figure out how to do it.

`<ComboBox ItemsSource="{Binding Filter}" SelectedItem="{Binding SelectedFilter}" Width="205" HorizontalContentAlignment="Left" />`

The text inside the box is correct, but as you can see in this screenshot, the dropdown box is still center aligned. What am I doing wrong?
![image](https://github.com/AvaloniaUI/Avalonia/assets/102810830/170daf4b-68bf-4db5-9bc8-d6e19d4b87bb)

**Comment by rabbitism** [comment]
It depends on your ItemTemplate.

**Reply by DarkMortainius** [reply, marked as answer]
Thank you, that was exactly what I needed!

```XAML
<ComboBox ItemsSource="{Binding Filter}" SelectedItem="{Binding SelectedFilter}" Width="205" HorizontalContentAlignment="Left">
    <ComboBox.ItemTemplate>
        <DataTemplate>
            <TextBlock Text="{Binding}" HorizontalAlignment="Left"></TextBlock>
        </DataTemplate>
    </ComboBox.ItemTemplate>
</ComboBox>
```
